server/models/chef.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const chefSchema = new Schema({
+  profilePicURL: String,
+  chefBio: {
+    tagLine: { type: Number, required: true },
+    fullBio: { type: Number, required: true },
+  },
+  cuisineSpecialty: { type: Number, required: true },
```

Comment:
Why is this a number? Also, are they limited to just one?

Reply:
That should definitely be a string, thanks for catching.  I had limited it to one because that's how I felt the mock up looked but I think its fine if they have more than.